packages/feature-toggle/README.md

```diff
@@ -0,0 +1,94 @@
+# @sap-ux/feature-toggle
+
+This module can be used for accessing and managing feature toggles in Fiori tools.## Definition of toggle
```

Comment:
Is "## Definition of toggle" supposed to go to the next line or is it to be deleted?

Reply:
Updated, thank you